### PR TITLE
SexyHub sitechange

### PR DESCRIPTION
**Pull request by SgtBatten**
Posters do not work... Can't figure out the right code.

Might add PAextras

### CODE REVIEW

**Comment by d2dyno** [comment]
Works for me in testing on MomXXX scenes. Thanks!

**Comment by fritz276** [comment]
Hmm. Not for me ?!

2019-08-01 19:02:11,273 (805b41300) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM1Cjg1MjI5czIKaWRzNDQKTW9teHh4IC0gQm9yZWQgaG9ybnkgaG91c2V3aWZlIGZ1Y2tzIG1hc3NldXJzNApuYW1lbnM0CnllYXJyMAo_
2019-08-01 19:02:11,274 (805b41300) :  DEBUG (runtime:49) - Received packed state data (1256 bytes)
2019-08-01 19:02:11,275 (805b41300) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM1Cjg1MjI5czIKaWRzNDQKTW9teHh4IC0gQm9yZWQgaG9ybnkgaG91c2V3aWZlIGZ1Y2tzIG1hc3NldXJzNApuYW1lbnM0CnllYXJyMAo_
2019-08-01 19:02:11,275 (805b41300) :  INFO (agentkit:957) - Searching for matches for {'year': None, 'id': '85229', 'name': 'Momxxx - Bored horny housewife fucks masseur'}
2019-08-01 19:02:11,276 (805b41300) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/85229/tree'
2019-08-01 19:02:11,279 (805b41300) :  INFO (__init__:46) - *******MEDIA TITLE****** Momxxx - Bored horny housewife fucks masseur
2019-08-01 19:02:11,279 (805b41300) :  INFO (__init__:53) - Getting Search Settings for: Momxxx - Bored horny housewife fucks masseur
2019-08-01 19:02:11,280 (805b41300) :  INFO (PAsearchSites:890) - mediaTitle w/ possible abbrieviation: Momxxx Bored horny housewife fucks masseur
2019-08-01 19:02:11,286 (805b41300) :  INFO (PAsearchSites:1071) - mediaTitle w/ possible abbrieviation fixed: Momxxx Bored horny housewife fucks masseur
2019-08-01 19:02:11,287 (805b41300) :  INFO (PAsearchSites:1086) - ^^^^^^^ siteID: 339
2019-08-01 19:02:11,288 (805b41300) :  INFO (PAsearchSites:1088) - ^^^^^^^ Shortening Title
2019-08-01 19:02:11,288 (805b41300) :  INFO (PAsearchSites:1089) - momxxx vs momxxx
2019-08-01 19:02:11,288 (805b41300) :  INFO (PAsearchSites:1093) - 1
2019-08-01 19:02:11,288 (805b41300) :  INFO (PAsearchSites:1148) - 7
2019-08-01 19:02:11,288 (805b41300) :  INFO (PAsearchSites:1148) - searchTitle (before date processing): Bored horny housewife fucks masseur
2019-08-01 19:02:11,288 (805b41300) :  INFO (__init__:63) - Search Title: Bored horny housewife fucks masseur
2019-08-01 19:02:11,289 (805b41300) :  INFO (__init__:72) - Bored%20horny%20housewife%20fucks%20masseur
2019-08-01 19:02:11,291 (805b41300) :  DEBUG (networking:166) - Requesting 'https://www.sexyhub.com/search/videos/Bored%20horny%20housewife%20fucks%20masseur'
2019-08-01 19:02:14,271 (805b41300) :  ERROR (networking:219) - Error opening URL 'https://www.sexyhub.com/search/videos/Bored%20horny%20housewife%20fucks%20masseur'
2019-08-01 19:02:14,272 (805b41300) :  CRITICAL (agentkit:1014) - Exception in the search function of agent named 'PhoenixAdult', called with keyword arguments {'year': None, 'id': '85229', 'name': 'Momxxx - Bored horny housewife fucks masseur'} (most recent call last):
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/agentkit.py", line 1007, in _search
    agent.search(*f_args, **f_kwargs)
  File "/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/__init__.py", line 483, in search
    results = PAsearchSites.networkSexyHub.search(results,encodedTitle,title,searchTitle,siteNum,lang,searchByDateActor,searchDate, searchSiteID)
  File "/Plex Media Server/Plug-ins/PhoenixAdult.bundle/Contents/Code/networkSexyHub.py", line 7, in search
    searchResults = HTML.ElementFromURL(PAsearchSites.getSearchSearchURL(siteNum) + encodedTitle)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/parsekit.py", line 482, in ElementFromURL
    method=method,
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/api/networkkit.py", line 67, in _http_request
    req = self._core.networking.http_request(url, *args, **kwargs)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 370, in http_request
    return HTTPRequest(self._core, url, data, h, url_cache, encoding, errors, timeout, immediate, sleep, opener, follow_redirects, method)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 141, in __init__
    self.load()
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 181, in load
    f = self._opener.open(req, timeout=self._timeout)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 444, in open
    response = meth(req, response)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 557, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 476, in error
    result = self._call_chain(*args)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 416, in _call_chain
    result = func(*args)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Versions/2/Python/Framework/components/networking.py", line 102, in http_error_302
    result = urllib2.HTTPRedirectHandler.http_error_302(self, req, fp, code, msg, headers)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 652, in http_error_302
    return self.parent.open(new, timeout=req.timeout)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 444, in open
    response = meth(req, response)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 557, in http_response
    'http', request, response, code, msg, hdrs)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 482, in error
    return self._call_chain(*args)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 416, in _call_chain
    result = func(*args)
  File "/usr/local/share/plexmediaserver-plexpass/Resources/Plug-ins-22929c8a2/Framework.bundle/Contents/Resources/Platforms/Shared/Libraries/urllib2_new.py", line 565, in http_error_default
    raise HTTPError(req.get_full_url(), code, msg, hdrs, fp)
HTTPError: HTTP Error 404: Not Found

**Comment by d2dyno** [comment]
You are on old code `https://www.sexyhub.com/search/videos/Bored%20horny%20housewife%20fucks%20masseur`

**Comment by SgtBatten** [comment]
Correct, old code and also appears not using the sceneID

**Comment by d2dyno** [comment]
Not sure if this is caused by the auto site determination change, or if the site needs to be updated - FitnessRooms looks to still be using old search url `https://www.fitnessrooms.com/search/videos/4360573`, even after I ran git pull to grab latest changes from this branch:

```
2019-08-01 20:53:48,184 (15065ad78700) :  INFO (__init__:46) - *******MEDIA TITLE****** Frs 19 07 04 4360573
2019-08-01 20:53:48,184 (15065ad78700) :  INFO (__init__:53) - Getting Search Settings for: Frs 19 07 04 4360573
2019-08-01 20:53:48,185 (15065ad78700) :  INFO (PAsearchSites:890) - mediaTitle w/ possible abbrieviation: Frs 19 07 04 4360573
2019-08-01 20:53:48,193 (15065ad78700) :  INFO (PAsearchSites:1066) - mediaTitle w/ possible abbrieviation fixed: FitnessRooms 19 07 04 4360573
2019-08-01 20:53:48,195 (15065ad78700) :  INFO (PAsearchSites:1081) - ^^^^^^^ siteID: 335
2019-08-01 20:53:48,195 (15065ad78700) :  INFO (PAsearchSites:1083) - ^^^^^^^ Shortening Title
2019-08-01 20:53:48,195 (15065ad78700) :  INFO (PAsearchSites:1084) - fitnessrooms  vs fitness rooms
2019-08-01 20:53:48,195 (15065ad78700) :  INFO (PAsearchSites:1143) - 3
2019-08-01 20:53:48,195 (15065ad78700) :  INFO (PAsearchSites:1143) - 7
2019-08-01 20:53:48,195 (15065ad78700) :  INFO (PAsearchSites:1143) - searchTitle (before date processing): 19 07 04 4360573
2019-08-01 20:53:48,196 (15065ad78700) :  INFO (__init__:63) - Search Title: 4360573
2019-08-01 20:53:48,196 (15065ad78700) :  INFO (__init__:67) - Search Date: 2019-07-04
2019-08-01 20:53:48,196 (15065ad78700) :  INFO (__init__:72) - 4360573
2019-08-01 20:53:48,197 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573
2019-08-01 20:53:48,197 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: 
2019-08-01 20:53:48,198 (15065ad78700) :  DEBUG (networking:166) - Requesting 'https://www.fitnessrooms.com/search/videos/4360573/1'
2019-08-01 20:53:48,412 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573
2019-08-01 20:53:48,412 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: 
2019-08-01 20:53:48,413 (15065ad78700) :  DEBUG (networking:161) - Fetching 'https://www.fitnessrooms.com/search/videos/4360573/1' from the HTTP cache
2019-08-01 20:53:48,424 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573
2019-08-01 20:53:48,424 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: 
2019-08-01 20:53:48,426 (15065ad78700) :  DEBUG (networking:161) - Fetching 'https://www.fitnessrooms.com/search/videos/4360573/1' from the HTTP cache
2019-08-01 20:53:48,437 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573
2019-08-01 20:53:48,437 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: 
2019-08-01 20:53:48,439 (15065ad78700) :  DEBUG (networking:161) - Fetching 'https://www.fitnessrooms.com/search/videos/4360573/1' from the HTTP cache
2019-08-01 20:53:48,454 (15065ad78700) :  DEBUG (runtime:88) - Sending packed state data (8492 bytes)
2019-08-01 20:53:48,454 (15065ad78700) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
2019-08-01 20:53:53,408 (15065ad78700) :  DEBUG (runtime:717) - Handling request GET /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjEwMzkxNHMyCmlkczUzCkZycyAxOSAwNyAwNCA0MzYwNTczLzItaG90LWJydW5ldHRlcy1zaGFyZS0xLWJpZy1kaWNrczQKbmFtZW5zNAp5ZWFycjAK
2019-08-01 20:53:53,410 (15065ad78700) :  DEBUG (runtime:49) - Received packed state data (8460 bytes)
2019-08-01 20:53:53,413 (15065ad78700) :  DEBUG (runtime:814) - Found route matching /:/plugins/com.plexapp.agents.phoenixadult/messaging/function/X0FnZW50S2l0OlNlYXJjaA__/Y2VyZWFsMQoxCmxpc3QKMApyMAo_/Y2VyZWFsMQoyCmRpY3QKZGljdAo2CnMyCmVuczQKbGFuZ2IxczYKbWFudWFsYjFzNwpwcmltYXJ5aTAKczcKdmVyc2lvbnIxCnM2Cmt3YXJnc3M1Ck1vdmllczEwCm1lZGlhX3R5cGUzCnM2CjEwMzkxNHMyCmlkczUzCkZycyAxOSAwNyAwNCA0MzYwNTczLzItaG90LWJydW5ldHRlcy1zaGFyZS0xLWJpZy1kaWNrczQKbmFtZW5zNAp5ZWFycjAK
2019-08-01 20:53:53,414 (15065ad78700) :  INFO (agentkit:957) - Searching for matches for {'year': None, 'id': '103914', 'name': 'Frs 19 07 04 4360573/2-hot-brunettes-share-1-big-dick'}
2019-08-01 20:53:53,414 (15065ad78700) :  DEBUG (networking:166) - Requesting 'http://127.0.0.1:32400/library/metadata/103914/tree'
2019-08-01 20:53:53,420 (15065ad78700) :  INFO (__init__:46) - *******MEDIA TITLE****** Frs 19 07 04 4360573/2-hot-brunettes-share-1-big-dick
2019-08-01 20:53:53,420 (15065ad78700) :  INFO (__init__:53) - Getting Search Settings for: Frs 19 07 04 4360573/2-hot-brunettes-share-1-big-dick
2019-08-01 20:53:53,420 (15065ad78700) :  INFO (PAsearchSites:890) - mediaTitle w/ possible abbrieviation: Frs 19 07 04 4360573/2 hot brunettes share 1 big dick
2019-08-01 20:53:53,429 (15065ad78700) :  INFO (PAsearchSites:1066) - mediaTitle w/ possible abbrieviation fixed: FitnessRooms 19 07 04 4360573/2 hot brunettes share 1 big dick
2019-08-01 20:53:53,430 (15065ad78700) :  INFO (PAsearchSites:1081) - ^^^^^^^ siteID: 335
2019-08-01 20:53:53,431 (15065ad78700) :  INFO (PAsearchSites:1083) - ^^^^^^^ Shortening Title
2019-08-01 20:53:53,431 (15065ad78700) :  INFO (PAsearchSites:1084) - fitnessrooms  vs fitness rooms
2019-08-01 20:53:53,431 (15065ad78700) :  INFO (PAsearchSites:1143) - 3
2019-08-01 20:53:53,431 (15065ad78700) :  INFO (PAsearchSites:1143) - 7
2019-08-01 20:53:53,431 (15065ad78700) :  INFO (PAsearchSites:1143) - searchTitle (before date processing): 19 07 04 4360573/2 hot brunettes share 1 big dick
2019-08-01 20:53:53,431 (15065ad78700) :  INFO (__init__:63) - Search Title: 4360573/2 hot brunettes share 1 big dick
2019-08-01 20:53:53,432 (15065ad78700) :  INFO (__init__:67) - Search Date: 2019-07-04
2019-08-01 20:53:53,432 (15065ad78700) :  INFO (__init__:72) - 4360573/2%20hot%20brunettes%20share%201%20big%20dick
2019-08-01 20:53:53,433 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573/2
2019-08-01 20:53:53,433 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: hot brunettes share 1 big dick
2019-08-01 20:53:53,434 (15065ad78700) :  DEBUG (networking:166) - Requesting 'https://www.fitnessrooms.com/search/videos/4360573/2/1'
2019-08-01 20:53:53,676 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573/2
2019-08-01 20:53:53,676 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: hot brunettes share 1 big dick
2019-08-01 20:53:53,678 (15065ad78700) :  DEBUG (networking:161) - Fetching 'https://www.fitnessrooms.com/search/videos/4360573/2/1' from the HTTP cache
2019-08-01 20:53:53,689 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573/2
2019-08-01 20:53:53,690 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: hot brunettes share 1 big dick
2019-08-01 20:53:53,691 (15065ad78700) :  DEBUG (networking:161) - Fetching 'https://www.fitnessrooms.com/search/videos/4360573/2/1' from the HTTP cache
2019-08-01 20:53:53,702 (15065ad78700) :  INFO (networkSexyHub:8) - SceneID: 4360573/2
2019-08-01 20:53:53,703 (15065ad78700) :  INFO (networkSexyHub:13) - Scene Title: hot brunettes share 1 big dick
2019-08-01 20:53:53,704 (15065ad78700) :  DEBUG (networking:161) - Fetching 'https://www.fitnessrooms.com/search/videos/4360573/2/1' from the HTTP cache
2019-08-01 20:53:53,720 (15065ad78700) :  DEBUG (runtime:88) - Sending packed state data (8492 bytes)
2019-08-01 20:53:53,721 (15065ad78700) :  DEBUG (runtime:924) - Response: [200] str, 320 bytes
```

**Comment by SgtBatten** [comment]
Nope i just missed it. Fixed

**Comment by d2dyno** [comment]
Resolved now. Are we waiting on anything else? LGTM

File: docs/sitelist.md [hunk 1]
```diff
@@ -631,7 +631,7 @@ The above is a reference list. Please see [the manualsearch doc](./manualsearch.
   - VIP Crew
   - We Live Together
   - Wives in Pantyhose
-+ #### SexyHub | Matching type: *[Limited](./manualsearch.md#limited-search)* - **Title only**
++ #### SexyHub | Matching type: *[Limited](./manualsearch.md#exact-match)* - **SceneID**
```
Comment (claygoldfinch) [inline docs/sitelist.md]: Could you update the text here to "exact" as well?

**Comment by ferengi82** [comment]
> Correct, old code and it will only match with sceneID now

is there any reason you don't use the search function on the site?
https://www.sexyhub.com/search?q=Stacy%20Cruz
seems to work fine

**Comment by SgtBatten** [comment]
Interesting. The actress search does seem to work. Title search does not though.

I'll see what I can do tomorrow

**Comment by SgtBatten** [comment]
@ferengi82 

Actually this is why its no good.

https://www.sexyhub.com/modelprofile/29772/nancy-a ------ she has 4 scenes

https://www.sexyhub.com/search?q=Nancy%20A ------ 1 scene of hers only

**Comment by claygoldfinch** [comment]
Agreed, the search functionality is just too unreliable for this style of site.

**Pull request merged**